src/main/resources/org/jenkinsci/test/acceptance/docker/fixtures/GitLabContainer/Dockerfile

```diff
@@ -0,0 +1,5 @@
+# Use the official GitLab Community Edition image as the base
+FROM gitlab/gitlab-ce:latest
```

Comment:
pin to a version

Reply:
Fixed, pinned to gitlab-ce:16.2.4-ce.0 and renovate set up to automate pr bumps to newer versions